frontend/notarial-blockchain-ui/src/App.tsx

```diff
@@ -1,28 +1,93 @@
 import './App.css';
 import Button from '@mui/material/Button';
+import axios from "axios";
+import {useState} from "react";
+import {CircularProgress} from "@material-ui/core";
+
+type ConnectionType = "NOT_CONNECTED" | "CONNECTING_ERROR" | "CONNECTED";
+type User = {
+  port?: string;
+  ip?: string;
+}
 
 const  App = () => {
+  const [connected, setConnected] = useState<ConnectionType>("NOT_CONNECTED");
+  const [loading, setLoading] = useState(false);
+  const [user, setUserData] = useState<User>({});
+  const [message, setMessage] = useState<String[]>([]);
```

Comment:
in the future maybe we can make an object here and add time when the event occurred?